### PR TITLE
Php2branch >tl1revbranch

### PR DESCRIPTION
**Pull request by php-2**
isetup modules task

### CODE REVIEW

File: portfolio/ifintech/v1/isetup/content/modules.php [hunk 1]
```diff
@@ -121,10 +121,11 @@
       $dbconn = new dbconn();
       $sql1= "SELECT modid FROM  " .dbname. "." .SETUP_DB_PREFIX ."imod "."WHERE status = 1";
      $conf_dtat=$dbconn->fnModData($sql1);
-     if($conf_dtat!=NULL)
+     if(!empty($conf_dtat))
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/isetup/content/modules.php]: $conf_dtat
what this parameter name means?

File: portfolio/ifintech/v1/isetup/content/modules.php [hunk 2]
```diff
@@ -184,14 +185,79 @@
...
+               }
+               if($("#label_iMOB_Android").hasclass("active")==true){
+              <?php
+                $sql =" INSERT INTO ".dbname. "." .SETUP_DB_PREFIX ."imod ";
+                $sql.="(modid,shortdesc,description,licensekey,status) VALUES (2, 'iMOB_Android_module','enable iMOB_Android',1,1)";
+                $dbconn->fnInsertData($sql);
+               ?>
+               }
+                if($("#label_iMOB_iOS").hasclass("active")==true){
+              <?php
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/isetup/content/modules.php]: why you add tag as <?php
the whole file is php in fact

File: portfolio/ifintech/v1/isetup/content/modules/modlog.php [hunk 3]
```diff
@@ -1,58 +1,15 @@
...
-
-
-    }
-
+global $userid; 
+$datenow = date("Y-m-d H:i:s");
+$sqllog =" INSERT INTO ".dbname. "." .SETUP_DB_PREFIX ."imod ";
+$sqllog.=" (userid,modid,newvalue,datacreation) VALUES ($userid,$modid,$modvalue,'$datenow')";
+if ($result){
+$sqlmod =" UPDATE ".MAIN_DB_PREFIX."imod ";
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/isetup/content/modules/modlog.php]: why still use MAIN_DB_PREFIX?

File: portfolio/ifintech/v1/isetup/content/config/config.php [hunk 4]
```diff
@@ -2,7 +2,7 @@
 /*
  * kamal.asran  dev.php2 22/1/2018
  * */
-require_once '../../constants.php' ;
+include_once '../../constants.php' ;
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/isetup/content/config/config.php]: where are the scripts for database, tables creation and records insertion?

Comment (php-2) [reply portfolio/ifintech/v1/isetup/content/config/config.php]: i created the database locally,i will write them when i back to this project ,

File: portfolio/ifintech/v1/isetup/content/modules.php (hunk 1 above)
Comment (php-2) [reply portfolio/ifintech/v1/isetup/content/modules.php]: it should be $mod_id i will rename it again

File: portfolio/ifintech/v1/isetup/content/modules.php (hunk 2 above)
Comment (php-2) [reply portfolio/ifintech/v1/isetup/content/modules.php]: the above statement is javascript and the next statment is php statment

File: portfolio/ifintech/v1/isetup/content/modules/modlog.php (hunk 3 above)
Comment (php-2) [reply portfolio/ifintech/v1/isetup/content/modules/modlog.php]: i will change it to SETUP_DB_PREFIX

**Pull request merged**